modules/backup-s3/module.go

```diff
@@ -18,52 +18,51 @@ import (
 	"github.com/pkg/errors"
 	"github.com/semi-technologies/weaviate/entities/modulecapabilities"
 	"github.com/semi-technologies/weaviate/entities/moduletools"
-	"github.com/semi-technologies/weaviate/modules/storage-aws-s3/s3"
+	"github.com/semi-technologies/weaviate/modules/backup-s3/s3"
 	"github.com/sirupsen/logrus"
 )
 
 const (
-	Name       = "storage-aws-s3"
-	AltName1   = "aws-s3"
-	AltName2   = "s3"
-	s3Endpoint = "STORAGE_S3_ENDPOINT"
-	s3Bucket   = "STORAGE_S3_BUCKET"
-	s3UseSSL   = "STORAGE_S3_USE_SSL"
+	Name       = "backup-s3"
+	AltName1   = "s3"
+	s3Endpoint = "BACKUP_S3_ENDPOINT"
+	s3Bucket   = "BACKUP_S3_BUCKET"
+	s3UseSSL   = "BACKUP_S3_USE_SSL"
 
 	// this is an optional value, allowing for
-	// the snapshot to be stored in a specific
+	// the backup to be stored in a specific
 	// directory inside the provided bucket.
 	//
 	// if left unset, the snapshot files will
 	// be stored directly in the root of the
 	// bucket.
-	s3SnapshotRoot = "STORAGE_S3_ROOT"
+	s3SnapshotRoot = "BACKUP_S3_PATH"
```

Comment:
s3SnapshotRoot -> s3Path?

Reply:
Good catch

Reply:
Oh yeah, actually, the backup -> snapshot name replacement should happen in another PR, since there are many other places where this occurs.

Reply:
But you are right about "root" -> "path". I will just change these now.

Reply:
fixed